crates/cnidarium-component/src/chain_state_read_ext.rs

```diff
@@ -0,0 +1,11 @@
+use anyhow::Result;
+use async_trait::async_trait;
+use cnidarium::StateRead;
+
+#[async_trait]
+pub trait ChainStateReadExt: StateRead {
+    async fn get_chain_id(&self) -> Result<String>;
+    async fn get_revision_number(&self) -> Result<u64>;
+    async fn get_block_height(&self) -> Result<u64>;
+    async fn get_block_timestamp(&self) -> Result<tendermint::Time>;
+}
```

Comment:
I don't think this should go in `cnidarium-component`, since this is really an IBC host chain interface, that should be named as such and go in the ibc crate, right?